cmake_toolchain/vita.cmake

```diff
@@ -77,21 +97,21 @@ macro(vita_create_self target source)
 
   ## SELF command
   separate_arguments(VITA_MAKE_FSELF_FLAGS)
-  add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${target}
-    COMMAND ${VITA_MAKE_FSELF} ${VITA_MAKE_FSELF_FLAGS} ${CMAKE_CURRENT_BINARY_DIR}/${sourcefile}.velf ${CMAKE_CURRENT_BINARY_DIR}/${target}
+  add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${self}
```

Comment:
how about just value compare&modify in here instead introducing new macro?
but it'll be an acceptable too.

Reply:
This would be the wrong place to put an `if`. Without introducing a new macro there would be no way for `make` builds to avoid circular dependencies.